src/fides/api/schemas/connection_configuration/connection_secrets_okta.py

```diff
@@ -0,0 +1,28 @@
+from typing import ClassVar, List
+
+from pydantic import Field
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class OktaSchema(ConnectionConfigSecretsSchema):
+    """Schema to validate the secrets needed to connect to Okta"""
+
+    org_url: str = Field(
+        title="Organization URL",
+        description="The URL of your Okta organization (e.g. https://your-org.okta.com)",
+    )
+    api_token: str = Field(
+        title="API Token",
+        description="The API token used to authenticate with Okta",
+        json_schema_extra={"sensitive": True},
+    )
+
+    _required_components: ClassVar[List[str]] = ["org_url", "api_token"]
```

Comment:
What's this used for? Doesn't look like it's accessed anywhere. By virtue of `org_url` and `api_token` not being `Optional` aren't they required?